examples/chef/chef_util.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import json
+import os
+import hashlib
+
+
+ci_allowlist = ['lighting-app.zap']
+cd_platforms_meta = {
+    'linux': {
+        'build_dir': 'linux/out',
+        'platform_label': 'linux_x86',
+    },
+    'esp32': {
+        'build_dir': 'esp32/build',
+        'platform_label': 'esp32-m5stack',
+    },
+    'nrfconnect': {
+        'build_dir': 'nrfconnect/build',
+        'platform_label': 'nrf-nrf52840dk',
+    },
+}
+
+
+def bundle(platform: str):
+    if platform == 'linux':
+        pass
+    if platform == 'esp32':
+        pass
+    if platform == 'nrfconnect':
+        pass
```

Comment:
No this was a placeholder. I have since removed it. To add a new platform to the pipelines (when CD is complete), all that would need to be done is:

Add to the metadata file
Add a per-platform bundle function

I think it would be ideal to keep the bundle functions isolated when the time comes, but could also go into chef itself.